composer.json

```diff
@@ -1,19 +1,25 @@
 {
-    "name": "ebln/parasite",
+    "name": "ebln/polyphonic-composing",
     "type": "library",
+    "description": "Experimenting on running different composer.jsons within the same codebase",
     "require": {
-        "php": "^8.1 || ^8.2",
+        "php": "^8.1",
+        "ebln/provoke-composer-conflict-foo": "^0.1.1",
+        "symfony/config": "^6.2",
         "symfony/console": "^6.2",
-        "ebln/provoke-composer-conflict-foo": "^0.1.0"
+        "symfony/dependency-injection": "^6.2",
+        "symfony/yaml": "^6.2"
+    },
+    "require-dev": {
+        "ergebnis/composer-normalize": "^2.29"
     },
     "autoload": {
         "psr-4": {
-            "Ebln\\ParasiteDemo\\": "src/"
+            "Ebln\\PolyphonicComposing\\": "src/"
         },
-        "exclude-from-classmap": ["parasite/"]
-    },
-    "require-dev": {
-        "ergebnis/composer-normalize": "^2.29"
+        "exclude-from-classmap": [
+            "src/Symbiont/**"
+        ]
```

Comment:
This does not work!
Possibly look into https://packagist.org/packages/mcaskill/composer-exclude-files